Dockerfile

```diff
@@ -14,7 +14,7 @@ ENV PATH="/opt/venv/bin:$PATH"
 
 RUN git clone https://github.com/sqlcipher/sqlcipher && \
     cd sqlcipher && \
-    git checkout v4.4.0 && \
+    git checkout v4.4.2 && \
```

Comment:
Hey, @kolya182 thanks for the PR. at the moment we have the application pinned to 4.4.0. 

I know that brew by default installs 4.4.2 but we have 4.4.0 pinned everywhere on our CI, so even our macos builds are build against 4.4.0. Rather than increasing the version it would be preferable to add information on how to checkout 4.4.0 version on OSX.

You can check the packaging.yml configuration

Reply:
https://github.com/rotki/rotki/pull/2512/commits/111bdc876658dddec671f0bd6b0cef0a386e3c2b